test/test_cuda.py

```diff
@@ -95,6 +95,13 @@ class TestCuda(TestCase):
     _do_cuda_non_default_stream = True
     FIFTY_MIL_CYCLES = 50000000
 
+    def setUp(self):
+        super(TestCuda, self).setUp()
+        self.autocast_lists = torch.cuda.amp._AutocastTestLists()
```

Comment:
Are you sure you really want every CUDA test case to be forced to run this prelude and postlude? It seems like a separate test class for AMP would make more sense.

Reply:
That's totally up to you guys.  The organization and placement of the tests is something I guessed at.